docusaurus/src/theme/NavbarItem.js

```diff
@@ -1,12 +1,25 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { useLocation } from '@docusaurus/router';
 import OriginalNavBarItem from '@theme-original/NavbarItem';
 
 const baseUrl = '/chat/docs/sdk';
 
 export default function NavbarItem(props) {
-  const { docsPluginId, items, label, type } = props;
+  const { docsPluginId, label, type } = props;
   const { pathname } = useLocation();
+  
+  if (label === 'SDK' && props.items.length) {
```

Comment:
hey, can you move the useMemo to outside the if block? [we shouldnt use hooks inside if blocks](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level). also, can you memoize the `.find` function as well? You can do the `length` checking inside the `useMemo` callback and return null or empty values in case length its 0.